Source/DiabloUI/credits_lines.cpp

```diff
@@ -187,15 +187,15 @@ const char *const CREDITS_LINES[] = {
 	"	Jackie Chan,Proposition 215, Rumiko Takahashi, Bunchy & Mason, ",
 	"	The Friends of Stan, The Zoo Crew, Team Hamro, Brazil 2001, RUSH, ",
 	"	Cornholio, THE BROS., Dar, Emeril Lagasse, Tom Waits, Ice-Cube, ",
-	N_("	The Strike Team Deflectors, Tony Levin, Big Jim and the Twins, "),
+	"	The Strike Team Deflectors, Tony Levin, Big Jim and the Twins, ",

```

Comment:
"and the Twins"?

Reply:
I wouldn't do so as it's a semi-personal line here.  (Also @Line 173)